examples/linear_model/plot_logistic_multinomial_rcv1.py

```diff
@@ -0,0 +1,101 @@
+"""
+=====================================================
+Multiclass logisitic regression on newgroups20
+=====================================================
+
+Comparison of multinomial logistic L1 vs one-versus-rest L1 logistic regression
+to classify documents from the newgroups20 dataset. Multinomial logistic
+regression yields more accurate results and is faster to train on the larger
+scale dataset.
```

Comment:
I would add a note:

Here we use the l1 sparsity that trims the weights of no to informative features to zero. This is good if the goal is to extract the strongly discriminative vocabulary of each class.

If the goal is to get the best predictive accuracy, it is better to use the non sparsity-inducing l2 penalty instead.